### PR TITLE
Delete inverse function and make a method on Dense

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

**Comment by btracey** [comment]
Fixes #178

File: mat64/dense_test.go [hunk 1]
```diff
@@ -1243,6 +1243,52 @@ func (s *S) TestOuter(c *check.C) {
...
+			}),
+			tol: 1e-14,
+		},
+	} {
+		var got Dense
+		err := got.Inverse(test.a)
+		if err != nil {
+			c.Errorf("Error computing inverse: %v", err)
+		}
+		if !EqualApprox(&got, test.want, test.tol) {
```
Comment (kortschak) [inline mat64/dense_test.go]: Can we have a wantEye.Mul(got, test.a) test? Also, I think equalApprox should be used for reasons I gave previously.

**Comment by kortschak** [comment]
LGTM with minor comment.

**Comment by btracey** [comment]
Failure is the same problem with Solve.

**Pull request merged**